StylableSwiftUI/Classes/StylistIdentifier.swift

```diff
@@ -246,31 +170,21 @@ extension StylistIdentifier.Component: ExpressibleByStringLiteral {
 
 extension StylistIdentifier {
 
-    /// Create an identifier representing `self` inside `identifier`
+    /// Create an identifier representing `self` inside `path`
     ///
     /// i.e. `"close".within("button") == 'button/close'`
     ///
-    public func within(_ identifier: StylistIdentifier?) -> StylistIdentifier {
-        guard let identifier = identifier else { return self }
+    public func within(_ path: Path?) -> StylistIdentifier {
+        guard let path = path else { return self }
```

Comment:
Don't think we need to do this since the (current) `within` method takes an optional

Reply:
Hah, investigated this and completely changed my mind about the optional in `Path.within(_:)`! This is used in a few places and it's far more readable just to deal with optionals internally, so I've put your isEmpty check in and left them as optionals! 

Removed the path=path line from this method - it's only really an optimisation if StylistIdentifier is a class - but it's a struct so I don't think we win anything by "optimising" here.